api/src/display.rs

```diff
@@ -42,12 +42,12 @@ fn calculate_checksum(data: &[u8]) -> [u8; 4] {
 /// encoded string, with a checksum prepended to it.
 impl printable::PrintableWrapper {
     /// Converts the proto to bytes and then encodes as b58
-    pub fn b58_encode(&self) -> Result<String, Error> {
+    pub fn b58_encode(&self) -> String {
```

Comment:
note: I first noticed this walking down some full-service calls that didn't seem like they should be fallible, so this will likely result in full-service needing some minor changes after this goes in.